src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_with_ppg.yaml

```diff
@@ -121,63 +121,34 @@ interactions:
       Connection:
       - keep-alive
       Content-Length:
-      - '1805'
+      - '1891'
       Content-Type:
       - application/json; charset=utf-8
       ParameterSetName:
       - --resource-group --name --location --dns-name-prefix --node-count --ssh-key-value
         --service-principal --client-secret --ppg
       User-Agent:
-      - python/3.6.10 (Linux-5.10.25-linuxkit-x86_64-with) msrest/0.6.21 msrest_azure/0.6.3
-        azure-mgmt-containerservice/11.2.0 Azure-SDK-For-Python AZURECLI/2.23.0 (DOCKER)
+      - python/3.8.0 (Windows-10-10.0.19041-SP0) msrest/0.6.21 msrest_azure/0.6.3
+        azure-mgmt-containerservice/11.0.0 Azure-SDK-For-Python AZURECLI/2.24.0
```

Comment:
Please re-record the live test with latest acs module dependency. Currently, we are using azure-mgmt-containerservice/11.2.0 with api version 2021-03-01, however this recording takes azure-mgmt-containerservice/11.0.0 with api version 2020-12-01.

Reply:
OK